### PR TITLE
feat: Adding solutions page layout in UI

### PR DESCRIPTION
**Pull request by Vandit1604**

### CODE REVIEW

File: src/css/cards.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    margin-top: 10px;
+    font-size: 18px;
+}
+
+.app-use-cases__card img {
+    width: 80px;
+    height: 80px;
+    margin-top: 20px;
+}
+
```
Comment (krisstern) [inline src/css/cards.css:38]:
```suggestion
  
```

Comment (krisstern) [inline src/css/cards.css]: Would be good to adhere to best practice by including a blank line in EOF for each file.

File: src/css/cards.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.app-use-cases__card h3 {
+    margin-top: 10px;
+    font-size: 18px;
+}
+
+.app-use-cases__card img {
+    width: 80px;
+    height: 80px;
+    margin-top: 20px;
+}
```
Comment (krisstern) [inline src/css/cards.css:37]:
```suggestion
}

```

**Comment by Vandit1604** [comment]
> Would be good to adhere to best practice by including a blank line in EOF for each file.
Hi @krisstern, Are suggesting this for the entire repository?

**Comment by krisstern** [comment]
> > Would be good to adhere to best practice by including a blank line in EOF for each file.

> Hi @krisstern, Are suggesting this for the entire repository?

Yes, for best practice in general.

**Review by krisstern — APPROVED** [review]
LGTM

**Pull request merged**